src/main/java/nl/thehyve/ocdu/WebSecurityConfig.java

```diff
@@ -36,4 +41,13 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
                 .inMemoryAuthentication()
                 .withUser("user").password("password").roles("USER");
     }
+
```

Comment:
In principle we should avoid committing commented-out code. If you want to keep some functionality for later which do not yet use - it will do no harm if it is uncommented (as long as the syntax is correct). 
Also mind that with correctly maintained git commits you can just safely delete stuff - you will always be able to retrieve it back from the history.